src/solc_0.7/common/Libraries/AddressUtils.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.7.1;
+
+library AddressUtils {
+    function toPayable(address _address) internal pure returns (address payable _payable) {
```

Comment:
There is now a new way to handle this in 0.7.  https://docs.soliditylang.org/en/v0.7.4/060-breaking-changes.html?highlight=payable#new-features

Reply:
fixed.